src/lib/fileUtilities.ts

```diff
@@ -10,10 +16,27 @@ const globAsync = promisify(glob);
  * @returns Arrays of files
  */
 export async function findAllFilesForPattern(
-  directory: string,
-  pattern: string
+   directory: string,
+   pattern: string
 ): Promise<string[]> {
-  const fileFindingExpression = directory + pattern;
-  return await globAsync(fileFindingExpression);
+   const fileFindingExpression = directory + pattern;
```

Comment:
You might consider `path.join` here to handle the `/` so it doesn't have to be part of the pattern (normally I don't think of globs as starting with a slash)

Reply:
Done